### DISCUSSION TITLE
To create a full circle button (style)

### DISCUSSION THREAD

**Discussion by paaljoachim** [WordPress/developer-blog-content#325] [Topic Ideas]
I have been struggling to create a full circle button with the Buttons block in this issue:
https://github.com/WordPress/gutenberg/issues/66081
Justin has been helping out giving feedback. Notice where I mix things up as I looked at the dev blog and partly found some of the needed information.

It would be nice to add it as a code snippet. Showing how to create a code snippet style using the needed code to do so.
Thank you. 

In general it would be great to add more code snippets into the Dev blog. Show how to setup a JSON file and code snippets where possible.

**Comment by bph** [comment]
Snippet approved, conversations continue on the issue: 
- #339

**Comment by paaljoachim** [comment]
Great. Thank you Birgit. I just did another test of the circle button and will continue some of the conversation directly in the issue.
